.vscode/cspell.json

```diff
@@ -637,6 +637,12 @@
         "Hhmms"
       ]
     },
+    {
+      "filename": "**/sdk/maps/**/*.cs",
+      "words": [
+        "Bbox"
+      ]
+    },
```

Comment:
We should remove this, because I expect the word "bbox" shouldn't exist in our code?

Reply:
Since we have this in generated files, should we change the spec, just to rename bbox to boundingBox ?
@dubiety

Reply:
Hi @jecmenicanikola ,

We can add directives in `sdk/maps/Azure.Maps.Search/src/autorest.md` like L#28-44 to manipulate the original swagger file. Please refer to [Property Rename](https://github.com/Azure/autorest/blob/main/docs/generate/directives.md#property-rename) to rename bbox to boundingBox

Reply:
@dubiety done